docs/configuration.asciidoc

```diff
@@ -2430,9 +2430,12 @@ Prepending an element with `(?-i)` makes the matching case sensitive.
 ==== `jms_listener_packages` (performance added[1.36.0])
 
 Defines which packages contain JMS MessageListener implementations for instrumentation.
-When set to a non-empty value, only the classes matching configuration will be instrumented.
+When empty (default), only the inner-classes or that have 'Listener' or 'Message' in their names are considered.
+
 This configuration option helps to make MessageListener type matching faster and improve application startup performance.
 
+Starting from version 1.43.0, the classes that are part of the 'application_packages' option are also included.
```

Comment:
how does that affect the default vs non-default?

Reply:
It does not affect it, it's just that classes that are part of the `application_packages` are also evaluated during class matching. In a sense it's just making the exclusion pre-filtering less restrictive than previously.

Reply:
so the default option will be inner-classes, classes with 'Listener' or 'Message' in the name, and all classes in application_packages?

Reply:
yes